pkg/manifest/os.go

```diff
@@ -287,6 +286,25 @@ func (p *OS) getContainerSources() []container.SourceSpec {
 	return p.OSCustomizations.Containers
 }
 
+func tomlPkgsFor(distro Distro) []string {
+	switch distro {
+	case DISTRO_EL7:
+		// nothing needs toml in rhel7
+		panic("no support for toml on rhel7")
+	case DISTRO_EL8:
+		// deprecated, needed for backwards compatibility (EL8 manifests)
+		return []string{"python3-pytoml"}
+	case DISTRO_EL9:
+		// older unmaintained lib, needed for backwards compatibility
+		return []string{"python3-toml"}
+	default:
```

Comment:
The CI fails right now https://gitlab.com/redhat/services/products/image-builder/ci/images/-/jobs/7618875295 with:
```
Failed to open file "/sys/fs/selinux/checkreqprot": Read-only file system
Traceback (most recent call last):
  File "/run/osbuild/bin/org.osbuild.bootc.install.config", line 6, in <module>
    import toml
ModuleNotFoundError: No module named 'toml'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/run/osbuild/bin/org.osbuild.bootc.install.config", line 8, in <module>
    import pytoml as toml
ModuleNotFoundError: No module named 'pytoml'
⏱  Duration: 0s
```
so this needs to be behaving like the previous code, at least fedora must return "python3-toml" for now

Reply:
yeah, let's keep the build roots identical for now.  Once osbuild/osbuild#1851 is merged and released, we can change our package selection and update the osbuild dependency to use the new module.

Reply:
It's been a while and the osbuild PR is merged/released, should we merge this now too?